Reneiski_Homework1/ReneiskiSelenium11/Tests/Tests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using NUnit.Framework;
+using ReneiskiSelenium11.Common.Extensions;
+using ReneiskiSelenium11.Data;
+using ReneiskiSelenium11.Data.Constants;
+using ReneiskiSelenium11.PageObjects;
+using ReneiskiSelenium11.PageObjects.Pages;
+
+namespace ReneiskiSelenium11.Tests
+{
+    public class DemoQATests : BaseTest
+    {
+
+        [Test]
+        public void FirstCheckBoxTest()
+        {
+            var check = new CheckBoxPage();
+
+            GenericPages.CheckBoxPage.GoToCheckBoxPage();
+
+            GenericPages.CheckBoxPage.ClickCheckBox();
```

Comment:
ClickCheckBox sounds too common because you have about 4 checkboxes on the page, it's better to make the test more specific, for example, click Home button, verify Home button is clicked